frameworks/moveos-stdlib/sources/event_queue.move

```diff
@@ -175,7 +175,38 @@ module moveos_std::event_queue {
         let event_queue_obj = borrow_mut_queue<E>(queue_id);
         internal_remove_expired_events(event_queue_obj);
     }
-    
+
+
+    public fun subscriber_info<E: copy + drop + store>(subscriber_obj: &Object<Subscriber<E>>):(u64, u64, u64) {
```

Comment:
Is it easier to call by passing `queue_name` instead of `subscriber_obj `?

Reply:
The arguments in the transaction are same, both are `ObjectID`

Reply:
I think there is not much difference between the two